frontend/src/metabase/query_builder/components/AlertListPopoverContent/AlertListItem.tsx

```diff
@@ -91,17 +92,29 @@ export const AlertListItem = ({
             />
           </li>
           {isAdmin && emailEnabled && emailChannel.recipients && (
-            <li className={cx(CS.ml3, CS.flex, CS.alignCenter)}>
+            <li
+              className={cx(CS.ml3, CS.flex, CS.alignCenter)}
+              aria-label="email recipient count"
+            >
               <Icon name="mail" className={CS.mr1} />
               {emailChannel.recipients.length}
             </li>
           )}
           {isAdmin && slackEnabled && (
-            <li className={cx(CS.ml3, CS.flex, CS.alignCenter)}>
-              <Icon name="slack" size={16} className={CS.mr1} />
-              {(slackChannel.details &&
-                slackChannel.details.channel.replace("#", "")) ||
-                t`No channel`}
+            <li
+              className={cx(CS.ml3, CS.flex, CS.alignCenter)}
+              aria-label="slack channel count"
```

Comment:
The previous item has a number in it corresponding to the count. Is there a similar number here? I just see the number 1 - is it really always one?

Maybe {t`Number of Slack channels`} for clarity

Reply:
You're only able to have 1 slack channel per alert, as far as I can tell